src/style.css

```diff
@@ -10,3 +10,63 @@
 	padding: 0;
 	text-transform: none;
 }
+
+.show-hide-toggle_master button {
+	font-size: .75em;
+	padding-right: 1.5em;
+	position: relative;
+	text-transform: uppercase;
+}
+
+.show-hide-toggle {
+	margin: 0;
+}
+
+.show-hide-toggle button {
+	border: 1px solid;
+	border-width: 1px 0;
+	border-radius: 0;
+	display: block;
+	margin:cc  0;
+	padding: .5em 1.5em .5em 0;
+	position: relative;
+	text-align: left;
+	transition: border .15s ease;
+	width: 100%;
+}
+
+.show-hide-toggle button:hover {
+	border-color: inherit;
+}
+
+.show-hide-toggle button[aria-expanded="true"] {
+	border-bottom: none;
+}
+
+.show-hide-toggle + .show-hide-panel + .show-hide-toggle button {
+	border-top: none;
+}
+
+.show-hide-toggle button::after,
+.show-hide-toggle_master button::after {
+	background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 10 10"><path fill="none" d="M0 0h10v10H0z"/><path fill="rgba(0,0,0,.67)" d="M10 4.69H5.31V0h-.62v4.69H0v.62h4.69V10h.62V5.31H10v-.62z"/></svg>');
+	content: "";
+	display: inline-block;
+	height: 1em;
+	position: absolute;
+	right: 0;
+	top: calc(50% - .5em);
+	width: 1em;
+	transition: transform .3s ease;
+}
+
+.show-hide-toggle button[aria-expanded="true"]::after,
+.show-hide-toggle_master button[aria-expanded="true"]::after {
+	transform: rotate(-135deg);
```

Comment:
This could look good with just `-45deg`, too - though I don't have a strong preference or think one is necessarily better than the other.

Reply:
I agree @philcable , as you know, I like things a little dramatic :)